src/script/main/index.tsx

```diff
@@ -32,10 +33,14 @@ import {doRedirect} from './app';
 
 import {SIGN_OUT_REASON} from '../auth/SignOutReason';
 import {Config} from '../Config';
+import {APIClient} from '../service/APIClientSingleton';
 import {StorageKey} from '../storage';
 
 document.addEventListener('DOMContentLoaded', async () => {
+  const apiClient = container.resolve(APIClient);
   const config = Config.getConfig();
+  await apiClient.useVersion(config.SUPPORTED_API_VERSIONS);
```

Comment:
This is the important fix, we need to set the version before initiating the core